.editorconfig

```diff
@@ -0,0 +1,126 @@
+############################### 
+# Core EditorConfig Options   # 
+############################### 
+root = true
+[*.{cs,csx,vb,vbx,md}] 
+indent_style = space 
+indent_size = 4 
+insert_final_newline = true 
+charset = utf-8
+end_of_line = lf
+[*.md] 
+trim_trailing_whitespace = false
+# Code files 
+[*.{cs,csx,vb,vbx}] 
+trim_trailing_whitespace = true
+############################### 
```

Comment:
why so many settings below this line, aren't the above ones enough?

Reply:
Those were added by VS, looks like VS has some additional settings, like possibility to make it prefer explicit types over var etc.